app.js

```diff
@@ -102,7 +103,8 @@ const LoadingView = React.createClass({
   render: function() {
     return (
         <div className={'loading'} onMouseEnter={this.enterView} onMouseLeave={this.leaveView}>
-          <a onClick={this.close} className={'close ' + this.state.hovered ? '': 'hidden'} />
+        <a onClick={this.close} data-tip data-for={'close-tip'} className={'close ' + this.state.hovered ? '': 'hidden'} />
```

Comment:
In [**activity-stream**](https://github.com/mozilla/activity-stream/blob/91763bb48083be73ea910fc9e02a7771e9ef6b21/content-src/components/Loader/Loader.js#L12) we use a module named [**classnames**](http://npm.im/classnames) which may be useful (or add unnecessary bloat, who knows).

```js
const classNames = require('classnames');

<a onClick={this.close}
   data-tip
   data-for={'close-tip'}
   className={classNames('close', {hidden: this.state.hovered})} />
```

... But cuts down on a bunch of inline ifs.